docs/book/src/user/quick-start.md

```diff
@@ -928,6 +928,8 @@ are also a few optional tunables if you'd like to change the OS or CIDRs used.
 export PROJECT_ID="2b59569f-10d1-49a6-a000-c2fb95a959a1"
 # The facility where you want your cluster to be provisioned
 export FACILITY="da11"
```

Comment:
/hold 

Should FACILITY be removed as part of this flow?

Ref from the Equinix Metal Provider README - https://github.com/kubernetes-sigs/cluster-api-provider-packet#ugrading-to-v07x

Reply:
Yes, it should be removed.

Reply:
It's removed